functions/src/utils/venue.js

```diff
@@ -6,8 +6,13 @@ const INVALID_SLUG_CHARS_REGEX = /[^a-zA-Z0-9]/g;
 const getVenueId = (name) =>
   name.replace(INVALID_SLUG_CHARS_REGEX, "").toLowerCase();
 
+const generateSlug = (name) => {
+  return name.replace(/\W/g, "").toLowerCase();
+};
+
```

Comment:
These should be a single function (ideally named `generateSlug()`) using `INVALID_SLUG_CHARS_REGEX ` instead of `/\W/g`

```suggestion
const getVenueId = (name) =>
  name.replace(INVALID_SLUG_CHARS_REGEX, "").toLowerCase();
};

```